addons/payment/views/payment_form_templates.xml

```diff
@@ -408,37 +407,66 @@
 
     <template id="payment.no_pms_available_warning">
         <div class="alert alert-warning mt-2">
-            <strong>No suitable payment option could be found.</strong>
-            <!-- === Availability report button (for debug) === -->
-            <t t-call="payment.availability_report_button"/>
+            <strong>No payment method available</strong>
             <br/>
-            If you believe that it is an error, please contact the website administrator.
+            <t t-if="request.env.is_system()">
```

Comment:
As suggested in https://github.com/odoo/odoo/pull/165203#discussion_r1722973536, you should merge this `t` with the `div` below.

Reply:
Done the changes